parser_with_temp.py

```diff
@@ -0,0 +1,258 @@
+        future usage"""
+        try:
+            self.base_url = site_url
+            self.logined_url = self.post(site_url, authentication_data).url
+        except requests.adapters.ConnectionError:
+            sys.exit(
+                'Connection issues. Check connection and try again')
+
+
+    def get_token(self):
+        """parses token from the response and creates session token property"""
+        try:
+            self.token = '&token=' + parse_qs(urlparse(
+                self.logined_url).query).get('token')[0]
+        except TypeError:
+            sys.exit(
+                'Login issues. Please enter properly username and password.')
+
+    def get_top_number_of_general_page(self):
+        """ sends request for the first page containing general table and grabs
+        the top number of pages containing general tables. Applies
+        top_page_number to the session"""
+        route = 'index.php?route=sale/order'
+        general_table_page_response = self.post(
+            self.base_url + route + self.token)
+        general_table_page = BeautifulSoup(
+            general_table_page_response.text.encode('utf-8'), "html.parser")
+        pages_urls = general_table_page.find('div', {'class': 'pagination'})
+        page_numbers = []
+        for url_item in pages_urls.findAll('a'):
+            page_numbers.append(int(re.split(r'=', (
+                url_item.get('href')))[-1]))
+        self.top_page_number = max(page_numbers)
+
+    def get_id_list(self):
+        """Iterates a list up to top_page_number, calling pages with general
+        tables and grabs all order id's into a list with all orders id's
+        :return a list with all orders numbers.
+        """
+        route = 'index.php?route=sale/order'
+        page = '&page='
+        _id_list = []
+        for page_number in range(self.top_page_number):
+            table_page = BeautifulSoup(self.post(
+                self.base_url + route + self.token + page + str(
+                    page_number + 1)).text.encode('utf-8'), "html.parser")
+            table_body = table_page.find('table', {'class': 'list'}).find(
+                'tbody')
+            for row in table_body.findAll('tr'):
+                if row.find('input').get('value') != '':
+                    _id_list.append(int(row.find('input').get('value')))
+        return _id_list
+
+
+def create_summary_dictionary(session, order_id):
+    """
+    Requests a page with full table, using session and order id.
+    Calls filling_order_table, wich returns detailed data of ordered goods.
+    Grabs necessary data into the summary dictionary.
+    :param session: Session object
+    :param order_id: int - number of desired order.
+    :return: summarized dictionary with full data about the order
+    """
+    route = 'index.php?route=sale/order/info'
+    order = '&order_id='
+    order_url = session.base_url + route + session.token + order + str(
```

Comment:
The best way is using page route and other additional data in requests, as "data" field in requests.post(...). But It doesn't works.